Training/interface/DataLoader_main.h

```diff
@@ -5,18 +5,26 @@
 #include "TROOT.h"
 #include "TLorentzVector.h"
 
-using namespace std;
-
-enum class CellObjectType {
-  PfCand_electron,
-  PfCand_muon,
-  PfCand_chargedHadron,
-  PfCand_neutralHadron,
-  PfCand_gamma,
-  Electron,
-  Muon
+template <typename T, typename Tuple>
+struct ElementIndex;
+
+template <typename T, typename... Args>
+struct ElementIndex<T, std::tuple<T, Args...>> {
+    static constexpr std::size_t value = 0;
+};
+
+template <typename T, typename U, typename... Args>
+struct ElementIndex<T, std::tuple<U, Args...>> {
+    static constexpr std::size_t value = 1 + ElementIndex<T, std::tuple<Args...>>::value;
 };
 
+using FeatureTuple = std::tuple<PfCand_electron_Features,
```

Comment:
perhaps `using FeatureTuple` should be generated in python to be consistent with the order in `CellObjectType`.